tools/lib/provision.py

```diff
@@ -319,9 +319,9 @@ def install_yum_deps(deps_to_install, retry=False):
         # Skip setup if it has been applied previously
         return
 
-    run(["sudo", "-H", "/usr/%s/bin/postgresql-%s-setup" % (postgres_dir, POSTGRES_VERSION), "initdb"])
+    run_as_root(["-H", "/usr/%s/bin/postgresql-%s-setup" % (postgres_dir, POSTGRES_VERSION), "initdb"])
```

Comment:
This one won't work, as `-H` was an argument to sudo.  I think probably we want a separate `sudo_args=['-H']` kwarg to run_as_root to handle this with something like this in the code.

```
if 'sudo_kwargs' in kwargs:
    sudo_cmd = ['sudo'] + kwargs['sudo_kwargs']
    del kwargs['sudo_kwargs']

```